all-your-base.md

```diff
@@ -0,0 +1,30 @@
+Implement general base conversion. Given a number on base **a**,
```

Comment:
I think it's _in_ (that's what you use lower down, and it sounds more natural to me).